tests/reused_shape_with_gradient_from_colr_1.svg

```diff
@@ -0,0 +1,24 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+  <defs>
+    <radialGradient id="g1" cx="60" cy="61.12" r="13.547" gradientUnits="userSpaceOnUse">
+      <stop offset="0.014" stop-color="#FFEB3B"/>
+      <stop offset="0.626" stop-color="#FCCD31"/>
+      <stop offset="1" stop-color="#FBC02D"/>
+    </radialGradient>
+    <radialGradient id="g2" cx="59.787" cy="60.693" r="13.547" gradientUnits="userSpaceOnUse">
+      <stop offset="0.005" stop-color="#81C784"/>
+      <stop offset="0.201" stop-color="#72BE76"/>
+      <stop offset="0.719" stop-color="#50A854"/>
+      <stop offset="1" stop-color="#43A047"/>
+    </radialGradient>
+    <radialGradient id="g3" cx="59.787" cy="60.48" r="13.547" gradientUnits="userSpaceOnUse">
+      <stop offset="0" stop-color="#E57373"/>
+      <stop offset="0.197" stop-color="#E56564"/>
+      <stop offset="0.718" stop-color="#E54542"/>
+      <stop offset="1" stop-color="#E53935"/>
+    </radialGradient>
+  </defs>
+  <path fill="url(#g1)" d="M77.60000000000001,64 Q77.60000000000001,60.26666666666667 75.73333740234375,57.17333221435547 Q73.86666666666667,54.080000000000005 70.82666778564453,52.26667022705078 Q67.78666666666668,50.45333333333334 64.05333333333334,50.45333333333334 Q60.21333333333334,50.45333333333334 57.17333221435547,52.26667022705078 Q54.13333333333334,54.080000000000005 52.26667022705078,57.17333221435547 Q50.400000000000006,60.26666666666667 50.400000000000006,64 Q50.400000000000006,67.73333333333335 52.26667022705078,70.82666015625 Q54.13333333333334,73.92000000000002 57.17333221435547,75.73333740234375 Q60.21333333333334,77.54666666666668 64.05333333333334,77.54666666666668 Q67.78666666666668,77.54666666666668 70.82666778564453,75.73333740234375 Q73.86666666666667,73.92000000000002 75.73333740234375,70.82666015625 Q77.60000000000001,67.73333333333335 77.60000000000001,64 Z"/>
```

Comment:
are we missing a rounding step somewhere?

Reply:
done